### DISCUSSION TITLE
Revision 1.7 Kicad project availability

### DISCUSSION THREAD

**Discussion by FussyKitten-dev** [will127534/StarlightEye#15] [General]
Hi, 

I recently purchased the StarlightEye and realized there are some changes from the v1.6 to v1.7, namely the inclusion of the gyro/accelerometer. I was curious if there were other changes to the schematic and layout to be aware about, mostly to get educated on any changes to the low noise power rails. Would it be possible to get an updated version of the kicad project pushed to the repo? Also, I made some changes to the IRFilter python code to also read the temperature from the TMP117 and get temp/gyro/accel data from the ICM42688. If that would be useful to anyone, let me know.

The camera board has been working well and has impressively low image noise at high gain values compared to the imx462 sensor I have been testing with. I haven't done a side-by-side comparison with the imx415 sensor I have but I expect it to beat that out for low light as well. Thanks for the good work.

Thanks,
FK

**Comment by will127534** [comment]
Thanks! Glad to hear that the board is working good for you.
I've added the V1.7 HW source file now https://github.com/will127534/StarlightEye/releases/tag/v1.7
There is no change to the analog rails, as the new IMU is hanging off the 1.8V IO rail.
